app/src/main/res/values/strings.xml

```diff
@@ -2140,14 +2147,87 @@ Please try again later"</string>
     <string name="local_library_manager_no_libraries_body">You don\'t have any Local libraries yet, try downloading one.</string>
     <string name="local_library_manager_download_status">Download status</string>
     <string name="local_library_manager_skip_downloading_dependencies">Skip downloading sub-dependencies</string>
-    <string name="signatures_title">Signatures</string>
     <string name="appearance">Appearance</string>
     <string name="appearance_description">Change the appearance of the Sketchware and make it your own!</string>
+    
     <string name="appearance_theme">Sketchware Theme</string>
     <string name="appearance_theme_system">System default</string>
     <string name="appearance_theme_light">Light theme</string>
     <string name="appearance_theme_dark">Dark theme</string>
-
+    <string name="appearance_theme_use_dynamic">Use dynamic theme</string>
```

Comment:
strings.xml are no longer maintained and cannot be easily used for translation by most users

Reply:
still better than hardcoding strings into code

Reply:
> still better than hardcoding strings into code

most of the extracted strings are only used once and take an extra step to change from there

Reply:
I agree with jbk.